functions/activate-batch-auto-background.ts

```diff
@@ -157,6 +157,15 @@ export async function handle(_?: any, db?: DynamoDB, contract?: any) {
           account.badLuckCount += balance
         }
 
+        // Enforce a cap to badLuckCount before it's committed back to
+        // the database.
+        const BAD_LUCK_COUNT_CAP = 26;
+        let finalBadLuckCount = account.badLuckCount;
+        if (finalBadLuckCount > BAD_LUCK_COUNT_CAP) {
+          finalBadLuckCount = BAD_LUCK_COUNT_CAP;
+        }
+        account.badLuckCount = finalBadLuckCount;
+
```

Comment:
You can do the check and assignment directly on `account.badLuckCount`, you don't need a temporary variable here

Also for BAD_LUCK_COUNT_CAP put it somewhere right at the top of the file (outside of the function), so it's easy to see/modify